README.md

```diff
@@ -1,11 +1,15 @@
-# [Ably](https://ably.com)
```

Comment:
We shouldn't be changing the README as part of this work. I'm guessing this content is supposed to appear in rubydoc API reference landing page, is there a way to change the content of that without changing the README?

Reply:
Fixed: created INTRO.md and reverted README.md.